app/models/question.rb

```diff
@@ -22,4 +22,8 @@ class Question < ActiveRecord::Base
   validates :number, presence: true, numericality: { only_integer: true }
 
   validates :question_type, presence: true, inclusion: {in: %w[short_answer multiple_choice true_false]}
+
+  def format_as_sms
```

Comment:
I think `formatted_for_sms` would be a better name. `format` by itself might imply that it mutates this object the `as` also sounds a bit awkward to me.

Maybe a better name is `formatted_question` like returns a formatted question.

I'll leave this to your discretion